pkg/kube/inject/testdata/inject/proxy-override-args.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hello
+spec:
+  selector:
+    matchLabels:
+      app: hello
+  template:
+    metadata:
+      labels:
+        app: hello
+    spec:
+      containers:
+      - name: hello
+        image: "fake.docker.io/google-samples/hello-go-gke:1.0"
+      - name: istio-proxy
+        image: auto
+        # Test that we can override a complex field like the command
+        # TODO this is broken since we append --concurrency to it
```

Comment:
I will fix this in another PR. We can merge that one first if we want, but its a pretty involved change

Reply:
https://github.com/istio/istio/pull/29380 has the fix

Reply:
Fix is merged